collaborative/IOU/heat_index.ipynb

```diff
@@ -162,218 +165,426 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9194b378-0a2c-4d2e-903e-039108305de9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# help for modifications \n",
+    "# 8pm Pacific is 3am UTC (0300 UTC), which is index 2\n",
+    "# 6am Pacific is 1pm UTC (1300 UTC), which is index 12\n",
+    "\n",
+    "night_subset = [2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12] # currently in UTC time\n",
+    "heatidx_hist_nighthours = heatidx_hist_hour.isel(time=heatidx_hist_hour.time.dt.hour.isin(night_subset))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ea99725a-44f0-4b2e-81a0-b4b73750a403",
+   "metadata": {},
+   "source": [
+    "### 3b) Sum the number of nights above a threshold of 80°F per day and per year\n",
+    "Like what we did above in Step 2a and 2b, we'll sum the nighttime Heat Index values for analysis. We'll use the same threshold as above, 80°F. But you can modify easily by setting this to any value of interest."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "6a845b92-d1c9-4b97-9936-98b6df649291",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# sum per day\n",
+    "num_heatidx_hist_nighthours = (heatidx_hist_nighthours >= hi_threshold).resample(time='1D').sum()\n",
+    "num_heatidx_hist_nighthours.name = 'Nighttime hours per day above Heat Index threshold of {}°F'.format(hi_threshold)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0b92fb40-c27e-4425-9599-b303d1a2a7ed",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# visualize\n",
+    "num_heatidx_hist_nighthours.hvplot.line(x='time', by='simulation')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a651e804-0c0e-42e0-bc91-830b4e6296d7",
    "metadata": {},
+   "outputs": [],
    "source": [
-    "Now, let's visualize the projected Heat Index:"
+    "# sum per year\n",
+    "num_heatidx_hist_nighthours_per_year = num_heatidx_hist_nighthours.groupby('time.year').sum('time')\n",
+    "num_heatidx_hist_nighthours_per_year.name = 'Nighttime hours per year above Heat Index threshold of {}°F'.format(hi_threshold)"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "bfd41f7e-1183-4f3d-80a3-00fe60e4eaa6",
+   "id": "9f9f5948-cf50-44f7-869d-cf7c3745dd3b",
    "metadata": {
     "tags": []
    },
    "outputs": [],
    "source": [
-    "ck.view(heatidx_proj_day)"
+    "# visualize\n",
+    "num_heatidx_hist_nighthours_per_year.hvplot.line(x='year', by='simulation')"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "b50dde60-9d24-495e-9d8b-9a140d5b05c5",
+   "id": "6ea6d794-e36c-449d-9b60-8f430cb6f7f5",
    "metadata": {},
    "source": [
-    "Compare the median historical and projected Heat Index values."
+    "We'll calculate the same steps for the projected future data, but won't visualize, just in case you need this information."
```

Comment:
```suggestion
    "We'll calculate the same steps for the projected future data, but won't visualize it here. We're just including it in case you need this information."
```